video/out/opengl/context_wayland.c

```diff
@@ -56,7 +81,39 @@ static void resize(struct ra_ctx *ctx)
 static void wayland_egl_swap_buffers(struct ra_ctx *ctx)
 {
     struct priv *p = ctx->priv;
+    struct vo_wayland_state *wl = ctx->vo->wl;
+
+    struct pollfd fds[2] = {
+        {.fd = wl->display_fd,     .events = POLLIN },
+        {.fd = wl->wakeup_pipe[2], .events = POLLIN },
+    };
+
+    wl_display_dispatch_pending(wl->display);
+
+    double vblank_time = (1/wl->current_output->refresh_rate)*1e6;
```

Comment:
Using "1e6 / ..." might be a little more elegant. Is refresh_rate already double?

Reply:
Yeah it is. 1e6/.. makes more sense.